Rakefile

```diff
@@ -11,14 +15,22 @@ bin = File.join(__dir__, "bin")
 
 Rake::ExtensionTask.new("rbs_extension")
 
-Rake::TestTask.new(:test => :compile) do |t|
+test_config = lambda do |t|
   t.libs << "test"
   t.libs << "lib"
   t.test_files = FileList["test/**/*_test.rb"].reject do |path|
     path =~ %r{test/stdlib/}
   end
 end
 
+Rake::TestTask.new(test: :compile, &test_config)
```

Comment:
What do you think about moving the check and the require here so everything is close together?

Reply:
Updated 👍